app/posts/manage-teacher-training-applications/2021-02-04-application-page-improvements.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Application page improvements
+description: Various improvements to the way we present work history and other parts of the application details page
+date: 2021-02-04
+tags:
+ - MN024
+ - MN025
+ - MN026
+---
+
+This iteration contains several improvements to the way we present the candidate’s application to providers.
+
+## Marking roles that are relevant to teaching
+
+We used to mark roles that involved working with children.
+
+But research shows that a role can be relevant to teaching even if it doesn’t involve working with children.
+
+We are instead going to ask candidates whether roles are relevant to teaching, so we changed the flag which can appear against work history items.
+
+## Removed job descriptions
+
+We used to ask candidates to enter descriptions of their roles. This created more work for candidates and also for providers, who had to read descriptions which were sometimes irrelevant.
+
+So we have removed the job descriptions for new applications. Instead a new ‘relevant to teaching’ flag indicates roles which providers might want to find out more about during an interview.
+
+Job descriptions should still be shown for old applications where the candidate did provide this information.
+
+## Making it easier to spot gaps in work
+
+Providers need to spot periods of time in which a candidate was not working or getting unpaid experience. 
+
+We have combined work experience and unpaid roles into one list, so that providers do not need to compare two separate lists. We mark unpaid experience so providers can distinguish between paid and unpaid roles.
+
+We have always indicated gaps but we now use the inset text component to make it easier to spot them.
```

Comment:
> We have always indicated gaps but we now use the inset text component to make it easier to spot them.

Is this true? I thought previously only gaps that had an explanation were shown?

Reply:
yes it’s true

Reply:
Interesting!